docs/contributing.md

```diff
@@ -180,6 +182,7 @@ In this example, only the relevant metadata items are shown:
         <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
         <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
         <meta property="dcterms:alternative">Title's base direction set to RTL</meta>
+        <meta property="belongs-to-collection">must</meta>
```

Comment:
Shouldn't this be omitted in the example? I thought we only wanted people to put this in if it's a should or may.

Reply:
I wanted to keep it, because it may not be clear for everyone how exactly to use this property (it is not frequently used). I have added a remark under the example making it clear that that specific entry could be omitted because it is the default.)